Bottom: ea488b7292c933804010e4e958a9b8fff109fe6d
Top:    ea488b7292c933804010e4e958a9b8fff109fe6d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-06-09 12:07:05 +0100

Add configure checks for GtkGLEext, add to CFLAGS and LIBS

*** TODO: Need separate tests for libgl and libglut etc.. ***















---


